contracts/mockup/VestingLogicMockup.sol

```diff
@@ -4,6 +4,15 @@ pragma experimental ABIEncoderV2;
 import "../governance/Vesting/VestingLogic.sol";
 
 contract VestingLogicMockup is VestingLogic {
+    /**
+     * @notice Max iteration for vesting withdrawal to prevent out of gas issue.
+     *
+     * @return max iteration value.
+     */
+    function getMaxVestingWithdrawIterations() public pure returns (uint256) {
```

Comment:
we can use now hardhat `hardhat_mine` RPC call to mine instantly as many blocks even at a predefined period of time.
pls consider using this `hh way` approach instead of adjusting the contracts design for mocking ;)
you can use network helpers for simplicity:
https://hardhat.org/hardhat-network-helpers/docs/reference#mine([blocks],-[options])